.travis.yml

```diff
@@ -27,7 +29,7 @@ stages:
 
 env:
   global:
-    - OpenBLAS_version=0.3.7
+    - OpenBLAS_version=0.3.8
```

Comment:
Technically it is `0.3.8.dev`. Perhaps in a follow-up PR we might clamp down on the "string in result" check in `openblas_support.py` to distinguish between the development branch and the stable release of `0.3.8`, but that's a minor point really.